### PR TITLE
add Dyson Protocol chain and assets

### PR DESCRIPTION
**Pull request by sybilsingleton**

### CODE REVIEW

File: dyson/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+          "denom": "dys",
+          "exponent": 0
+        }
+      ],
+      "base": "dys",
+      "name": "Dys",
+      "display": "dys",
+      "symbol": "DYS",
+      "logo_URIs": {
+        "png": "https://gitlab.com/dysonproject/dyson/-/raw/master/frontend/static/img/dyson-logo.png"
```
[the remark this answers is not in the document]
Comment (sybilsingleton) [reply dyson/assetlist.json]: Sure thing, just added it now

File: dyson/chain.json [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        "address": "https://dys-api.dyson.lol:443",
+        "provider": "lol"
+      },
+      {
+        "address": "https://dys-api.dysonprotocol.com:443",
+        "provider": "dysonprotocol"
+      }
+    ],
+    "grpc": [
+      "https://dys-grpc.dyson.lol:443"
```
Comment (JeremyParish69) [inline dyson/chain.json:88]: I think this needs to be in an endpoint or api object

Comment (sybilsingleton) [reply dyson/chain.json:88]: It's not the grpc endpoint?
```
$ grpcurl dys-grpc.dyson.lol:443 list
cosmos.auth.v1beta1.Query
cosmos.authz.v1beta1.Query
cosmos.bank.v1beta1.Query
cosmos.base.reflection.v1beta1.ReflectionService
cosmos.base.reflection.v2alpha1.ReflectionService
cosmos.base.tendermint.v1beta1.Service
cosmos.distribution.v1beta1.Query
cosmos.evidence.v1beta1.Query
cosmos.feegrant.v1beta1.Query
cosmos.gov.v1.Query
cosmos.gov.v1beta1.Query
cosmos.group.v1.Query
cosmos.mint.v1beta1.Query
cosmos.params.v1beta1.Query
cosmos.slashing.v1beta1.Query
cosmos.staking.v1beta1.Query
cosmos.tx.v1beta1.Service
cosmos.upgrade.v1beta1.Query
dyson.Query
grpc.reflection.v1alpha.ServerReflection
ibc.applications.interchain_accounts.controller.v1.Query
ibc.applications.interchain_accounts.host.v1.Query
ibc.applications.transfer.v1.Query
ibc.core.channel.v1.Query
ibc.core.client.v1.Query
ibc.core.connection.v1.Query
names.Query
```

Comment (sybilsingleton) [reply dyson/chain.json:88]: I used this as a reference: https://github.com/cosmos/chain-registry/blob/master/cosmoshub/chain.json#L366
```
$ grpcurl cosmoshub-grpc.lavenderfive.com:443 list
cosmos.auth.v1beta1.Query
cosmos.authz.v1beta1.Query
cosmos.bank.v1beta1.Query
cosmos.base.reflection.v1beta1.ReflectionService
cosmos.base.reflection.v2alpha1.ReflectionService
cosmos.base.tendermint.v1beta1.Service
cosmos.distribution.v1beta1.Query
cosmos.evidence.v1beta1.Query
cosmos.feegrant.v1beta1.Query
cosmos.gov.v1beta1.Query
cosmos.mint.v1beta1.Query
cosmos.params.v1beta1.Query
cosmos.slashing.v1beta1.Query
cosmos.staking.v1beta1.Query
cosmos.tx.v1beta1.Service
cosmos.upgrade.v1beta1.Query
grpc.reflection.v1alpha.ServerReflection
ibc.applications.interchain_accounts.controller.v1.Query
ibc.applications.interchain_accounts.host.v1.Query
ibc.applications.transfer.v1.Query
ibc.core.channel.v1.Query
ibc.core.client.v1.Query
ibc.core.connection.v1.Query
router.v1.Query
tendermint.liquidity.v1beta1.Query
```

Comment (sybilsingleton) [reply dyson/chain.json:88]: Nevermind, I get it now, it was the schema failing

Comment (sybilsingleton) [reply dyson/chain.json]: Ok rebased from master again, had a small snafu

File: dyson/assetlist.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (JeremyParish69) [reply dyson/assetlist.json:18]: I thought there was an image added here (in this repo) earlier, now it's gone again?

File: dyson/chain.json [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    "evm-http-jsonrpc": []
+  },
+  "explorers": [
+    {
+      "kind": "ping.pub",
+      "url": "https://explorer.dys.dysonprotocol.com/dyson/",
+      "tx_page": "https://explorer.dys.dysonprotocol.com/dyson/tx/${txHash}"
+    },
+    {
+      "kind": "ping.pub",
```
Comment (JeremyParish69) [inline dyson/chain.json:102]: also, should probably change these "kind"/providers? as 'lol' and 'dysonprotocol' maybe?

Comment (sybilsingleton) [reply dyson/chain.json]: thanks, I wasn't certain what to put there

File: dyson/assetlist.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (sybilsingleton) [reply dyson/assetlist.json]: oops the rebase was done poorly, readded it

File: dyson/chain.json [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+      {
+        "address": "https://dys-grpc.dyson.lol:443",
+        "provider": "lol"
+      }
+    ],
+    "evm-http-jsonrpc": []
+  },
+  "explorers": [
+    {
+      "dysonprotocol": "dysonprotocol",
```
Comment (JeremyParish69) [inline dyson/chain.json:97]: sorry, it looks like you accidentally changed the property name rather than just the value. Could you please correct that?

Comment (sybilsingleton) [reply dyson/chain.json]: fixed!

**Comment by sybilsingleton** [comment]
@JeremyParish69 Hi, is there something I need to do to trigger the checks?

**Review by JeremyParish69 — APPROVED** [review]

**Pull request merged**